src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -162,6 +185,60 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example of allowed package visible members:
+ * </p>
+ * <pre>
+ * public class MyClass
+ * {
+ * private int myPrivateField1;
+ * public long serialVersionUID = 123456789L;
+ * public static final int MY_CONSTANT = 42;
+ * int myPrivateField3;
+ * protected String myProtectedField; // Violation, protected visibility is not allowed
+ * public final int MY_CONSTANT2 = 42; // Violation, public immutable fields are not allowed
+ * public int NOT_CONSTANT = 42; // Violation, not static final, immutable, matching the pattern
```

Comment:
Try to keep violation messages indented at the same level:
```java
 int myPrivateField3;                // Violation, must have a visibility modifier
 protected String myProtectedField;  // Violation, protected visibility is not allowed
 public final int MY_CONSTANT2 = 42; // Violation, public immutable fields are not allowed
```
If line length doesn't permit, try shortening variable names, changing that particular example also works.

Reply:
Even i changed the variable names still more than 100 characters. 
I will, remove the whole comment and just leave "Violation"

Reply:
No, please do not remove the violation message, having a violation message is a must. You can change grouping to accommodate violation messages, use `// violation above, ....`, `// violation below, ....` pattern too (last preference).

Also please use `// violation` and not `// Violation`.

Reply:
Please check the updates.
For the failure, it seems like something in the CI, not from me, right?